db/migrate/20150818184658_create_tenant_quota.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTenantQuota < ActiveRecord::Migration
+  def change
+    create_table :tenant_quotas do |t|
+      t.belongs_to :tenant, type: :bigint
+
+      t.string     :name
+      t.string     :unit
+      t.float      :value
+
```

Comment:
@gtanzillo 
Do we need a description of the item that the UI can use for display?